examples/readme-opentelemetry.rs

```diff
@@ -5,15 +5,11 @@ fn main() {
     use opentelemetry::trace::TracerProvider;
     use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
 
-    let exporter = opentelemetry_stdout::SpanExporter::builder()
-        .with_writer(std::io::sink())
-        .build();
```

Comment:
This is supposed to copy the exmaple inside README.md so that we can be sure that the example there works. Can you please also change it there?